operator/src/main/java/oracle/kubernetes/operator/wlsconfig/WlsDomainConfig.java

```diff
@@ -510,22 +513,48 @@ public WlsDomainConfig withCluster(WlsClusterConfig clusterConfig) {
 
   @Override
   public String toString() {
-    return "WlsDomainConfig{"
-        + "name='"
-        + name
-        + '\''
-        + ", adminServerName='"
-        + adminServerName
-        + '\''
-        + ", configuredClusters="
-        + configuredClusters
-        + ", servers="
-        + servers
-        + ", serverTemplates="
-        + serverTemplates
-        + ", wlsMachineConfigs="
-        + wlsMachineConfigs
-        + '}';
+    return new ToStringBuilder(this)
+        .append("name", name)
+        .append("adminServerName", adminServerName)
+        .append("configuredClusters", configuredClusters)
+        .append("servers", servers)
+        .append("serverTemplates", serverTemplates)
+        .append("wlsMachineConfigs", wlsMachineConfigs)
```

Comment:
FYI - I suspect wlsMachineConfigs is a dead code path - the operator now gets config from the introspector, and I don't think the introspector returns machine configs...